src/main/java/com/google/api/codegen/config/DiscoveryMethodModel.java

```diff
@@ -229,4 +258,37 @@ public boolean hasReturnValue() {
     inputFields = fieldsBuilder.build();
     return inputFields;
   }
+
+  @Override
+  public Iterable<FieldModel> getOutputFields() {
+    if (outputFields != null) {
+      return outputFields;
+    }
+
+    List<FieldModel> outputField;
+    if (method.response() != null && !Strings.isNullOrEmpty(method.response().reference())) {
+      FieldModel fieldModel = new DiscoveryField(method.response().dereference(), null);
+      outputField = Lists.newArrayList(fieldModel);
+    } else {
+      outputField = new ArrayList<>();
+    }
+    outputFields = ImmutableList.copyOf(outputField);
```

Comment:
Why not just make an ImmutableList from the beginning instead of converting?

Reply:
Done.